docs/application/dotnet/guides/nui/styling-controls-with-xaml.md

```diff
@@ -0,0 +1,200 @@
+The NUI also defines the `ViewStyle` derived classes for components such as  `ImageView`, `TextLabel` and `TextField`:
+
+```csharp
+ImageViewStyle swtichImageStyle = new ImageViewStyle
+{
+    ResourceURL = new StringSelector
+    {
+        Normal = "controller_switch_bg_off.png",
+        Selected = "controller_switch_bg_on.png",
+        Disabled = "controller_switch_bg_off_dim.png",
+        DisabledSelected = "controller_switch_bg_on_dim.png"
+    }
+};
+
+TextLabelStyle redTextStyle = new TextLabelStyle()
+{
+  TextColor = Color.Red,
+  PointSize = 20
+};
+
+TextFieldStyle textStyle = new TextFieldStyle()
+{
+  EnableCursorBlink = False
+};
+```
+
+## Define theme using XAML
+
+It is better to use XAML theme files in advanced applications to define UI style for NUI widgets. With XAML files, it is easier to share styles between different applications. From the application architecture point of view, XAML allows for better separation of view definitions from application logic.
+
+### XAML example
+
+ The basic text styles defined in the earlier section can be implemented using XAML as follows:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<Theme
+  xmlns="http://tizen.org/Tizen.NUI/2018/XAML"
+  xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml">
+
+  <TextLabelStyle x:Key="redTextStyle" TextColor="Red" PointSize="20" />
+</Theme>
+```
+
+### Handle component states
+
+The creation of a `Selector` object defines the various states of a component. In `Selector` constructor, individual states must be specified:
+
+```csharp
+Selector selector = new Selector<Color>()
+{
+  Normal = Color.Black,
+  Pressed = Color.Blue,
+  Disabled = Color.Red
+};
+```
+
+The equivalent XAML code is as follows:
+
+```xml
+<Selector x:TypeArguments="Color" Normal="Black" Pressed="Blue" Disabled="Red"/>
+```
+
+The selectors work when the control state is enabled in a `View`. By default, the control state is enabled for all controls in the `Tizen.NUI.Components` namespace and not for the controls that belong to `Tizen.NUI.BaseComponents` namespace, such as `View` and `TextLabel`. If you want to enable a control state for them, set the `EnableControlState` property to `true`:
+
+```xml
+<ViewStyle x:Key="viewColoredByTouch" EnableControlState="true">
+  <ViewStyle.BackgroundColor>
+      <Selector x:TypeArguments="Color" Normal="Black" Pressed="Blue" Disabled="Red"/>
+  </ViewStyle.BackgroundColor>
+</ViewStyle>
+```
+
+### Load theme in NUI application
+
+To use the theme implemented in XAML, follow these steps: 
+
+1. Create a theme object as follows:
+     ```csharp
+    Theme theme = new Theme("PATH") \\ where PATH is a full path to the XAML file in a project.
+     ```
+2. Use `ThemeManager` to apply the theme as follows:
+    ```csharp
+   ThemeManager.ApplyTheme(theme)
```

Comment:
Why don't we have ; at the end of the code line, is it just because every developer knows there must be one and thus is ignored?

I think we must keep ; 
Line 114 & 110. 
Kindly verify .